packages/core/types/zapier.custom.d.ts

```diff
@@ -95,6 +95,14 @@ export interface HttpRequestOptions {
   timeout?: number;
   url?: string;
   skipThrowForStatus?: boolean;
+
+  /**
+   * Custom options for the request. These do not affect the request
+   * itself, but can be used to pass data to middleware. For example,
+   * this is recommended for things like `prefixErrorMessage` fields
+   * etc.
+   */
+  customOptions?: Record<string, any>;
```

Comment:
This isn't an actual option provided by `z.request()`, so I'm not sure if we should include this. If we added this, we'd have to support it forever until we can make it a breaking change.

@tkcranny is there another way for you to get around this?